content/applications/sales/rental.rst

```diff
@@ -0,0 +1,93 @@
+======
+Rental
+======
+
+**Odoo Rental** is a comprehensive solution to manage your rentals.
+
+From a single view, you can send out quotations, confirm orders, schedule rentals, register when
+products are picked up and returned, and invoice your customers.
+
+.. seealso::
+   - `Odoo Rental: product page <https://www.odoo.com/app/rental>`_
+
+Configuring rental pricing
+==========================
+
+Go to :menuselection:`Rental --> Products`, select or create a product, and click on the
+product's *Rental tab*. Under *Rental Pricing*, click on *Add a price*. Then you choose a *Unit*
+of time (hours, days, weeks, or months), a *Duration*, and a *Price*. You can add as many price
+lines as necessary, usually to give out discounts for longer rental durations.
+
+.. image:: rental/rental-pricing-example.png
+   :align: center
+   :alt: Example of rental pricing configuration in Odoo Rental
+
+.. tip::
+   Under *Reservations*, you can add fines for any *Extra Hour* or *Extra Day*. You can also set a
+   *Security Time*, expressed in hours, to make the product temporarily unavailable between two
+   rental orders.
+
+.. note::
+   If you want to rent a product created outside the Rental app, do not forget to tick *Can be
+   Rented* under the product's name. By default, this option is ticked when you create a product
+   directly from the Rental app.
+
+Computing rental pricing
+------------------------
+
+Odoo always uses two rules to compute the price of a product when you create a rental order:
+
+#. Only one price line is used.
+#. The cheapest line is selected.
+
+Consider the following rental pricing configuration for a product:
+
+- 1 day: $100
+- 3 days: $250
+- 1 week: $500
+
+A customer wants to rent this product for 8 days. After an order is created, Odoo selects the
+second line as this is the cheapest option. The customer has to pay 3 times '3 days' to
+cover the rental's 8 days, for a total of $750.
+
```

Comment:
I think the story flows much better here. I still think it could be worthwhile to mention either here or in the mention of the price lines that your customers will get the best prices when more price lines are added to your database.

Reply:
I agree. However, when you start explaining all that, it emphasizes that the way Odoo computes the price is silly. IMO, there should be at least an option to mix price lines to ensure customers get the best prices (I guess devs might have faced some technical difficulties in implementing that?).
Anyway, I didn't want to put too much focus on that, just explain the way it works.

Reply:
That makes sense to me! I was thinking that it could be a problem too if we dove too far into that functionality.